docs/basics/101-125-config.rst

```diff
@@ -0,0 +1,327 @@
+   For example, it lists the repository as a
+   "version 5 repository", and gives the dataset its own Git-annex
+   UUID. While the "annex-uuid" [#f4]_ looks like yet another cryptic
+   random string of characters, you have seen a UUID like this before:
+   A :command:`git annex whereis` displays information about where the
+   annexed content in a dataset is with these UUIDs.
+   This section also specifies the supported backends in this dataset.
+   If you have read the hidden section in the section
+   :ref:`symlink` you will recognize the name "MD5E". This is the
+   hash function used to generate the annexed files keys and thus
+   paths in the object tree. All backends specified in this file (it
+   can be a list) can be used to hash your files.
+
+   You may recognize the third part of the configuration, the subsection
+   ``"recordings/longnow"`` in the section ``submodule``.
+   Clearly, this is a reference to the ``longnow`` podcasts
+   we installed as a subdataset. The name *submodule* is Git
+   terminology, and describes a Git repository inside of
+   another Git repository -- just like
+   the super- and subdataset principles you discovered in the
+   section :ref:`nesting`. When you install a DataLad dataset
+   as a subdataset, it gets *registered* in this file.
+   For each subdataset, an individual submodule entry
+   will store the information about the subdataset's
+   ``--source`` or *origin* (the "url").
+   Thus, every subdataset (and sub-subdataset, and so forth) in your dataset
+   will be listed in this file.
+   If you want, go back to section :ref:`installds` to see that the
+   "url" is the same URL we installed the longnow dataset from, and
+   go back to section :ref:`sharelocal1` to remind yourself of
+   how installing a dataset with subdatasets looked and felt like.
+
+   Another interesting part is the last section, "remote".
+   Here we can find the :term:`sibling` "roommate" we defined
+   in :ref:`sibling`. The term :term:`remote` is Git-terminology and is
+   used to describe other repositories or DataLad datasets that the
+   repository knows about and tracks.
+   This file, therefore, is where DataLad *registered* the sibling
+   with :command:`datalad siblings add`, and thanks to it you can
+   collaborate with your room mate.
+   Note the *path* given as a value to the ``url`` variable. If at any point
+   either your superdataset or the remote moves on your file system,
+   the association between the two datasets breaks -- this can be fixed by adjusting this
+   path, and a demonstration of this is in section (Todo: link file system management).
+   `fetch` contains a specification which parts of the repository are
+   updated -- in this case everything (all of the branches).
+   Lastly, the ``annex-ignore = false`` configuration allows Git-annex
+   to query the remote when it tries to retrieve data from annexed content.
+
+``.git/config`` versus other (configuration) files
```

Comment:
This could be misleading. Also global and system-wide config are not shared with a dataset. They just happen to be in effect on a system, or for a user.